storage/test/transfer.test.js

```diff
@@ -191,78 +172,55 @@ describe('storage:transfer', function () {
         var value = 'DISABLED';
```

Comment:
What is the point of these variables if we're manipulating `options` and not them?